code/src/components/Form.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from "react";
+import "Css/app.css";
+
+export const Form = () => {
+  const [approved, setApproved] = useState("ok i guess")
+  const [text, setText] = useState("")
+  const [submit, setSubmit] = useState(false)
+  const [question, setQuestion] = useState(1)
+  const [select, setSelect] = useState("nothing")
+  const [rate, setRate] = useState("5")
```

Comment:
This can be a boolean instead of string
```suggestion
  const [rate, setRate] = useState(5)
```